versions/3.1.1.md

```diff
@@ -191,6 +191,8 @@ When parsing an OAD, JSON or YAML objects are parsed into specific Objects (such
 
 If the same JSON/YAML object is parsed multiple times and the respective contexts require it to be parsed as _different_ Object types, the resulting behavior is _implementation defined_, and MAY be treated as an error if detected.  An example would be referencing an empty Schema Object under `#/components/schemas` where a Path Item Object is expected, as an empty object is valid for both types.  For maximum interoperability, it is RECOMMENDED that OpenAPI Description authors avoid such scenarios.
 
+#### <a name="resolvingImplicitConnections"></a>Resolving Implicit Connections
+
```

Comment:
This looks like an empty H4 ... is that what was intended? It looks very odd.

Reply:
It's to keep the markdown validator from complaining about broken links.  That is being added in another PR.  I have like 6 different PRs that cross-reference each other and there's no good way to do this.  Of the three ways I've tried, this was the only one that @ralfhandl didn't object to.

It will all get sorted as we merge and resolve conflicts.